controller/model/create_terminator_cmd.go

```diff
@@ -0,0 +1,103 @@
+package model
+
+import (
+	"github.com/michaelquigley/pfxlog"
+	"github.com/openziti/edge/controller/persistence"
+	"github.com/openziti/edge/edge_common"
+	"github.com/openziti/edge/pb/edge_cmd_pb"
+	"github.com/openziti/fabric/controller/command"
+	"github.com/openziti/fabric/controller/network"
+	"github.com/pkg/errors"
+	"go.etcd.io/bbolt"
+	"strings"
+)
+
+type CreateEdgeTerminatorCmd struct {
+	Env    Env
+	Entity *network.Terminator
+}
+
+func (self *CreateEdgeTerminatorCmd) Apply() error {
+	createCmd := &command.CreateEntityCommand[*network.Terminator]{
+		Creator:        self.Env.GetManagers().Terminator,
+		Entity:         self.Entity,
+		PostCreateHook: self.validateTerminatorIdentity,
+	}
+	return self.Env.GetManagers().Terminator.ApplyCreate(createCmd)
+}
+
+func (self *CreateEdgeTerminatorCmd) validateTerminatorIdentity(tx *bbolt.Tx, terminator *network.Terminator) error {
+	session, err := self.getTerminatorSession(tx, terminator, "")
+	if err != nil {
+		return err
+	}
+
+	if terminator.GetIdentity() == "" {
+		return nil
+	}
+
+	identityTerminators, err := self.Env.GetStores().Terminator.GetTerminatorsInIdentityGroup(tx, terminator.GetId())
+	for _, otherTerminator := range identityTerminators {
+		otherSession, err := self.getTerminatorSession(tx, otherTerminator, "sibling ")
```

Comment:
just confirming the space here in "sibling " intentional?

Reply:
Yeah, used for error messages and it can be either blank or a string. The space just means the target method doesn't need to figure out if a space is necessary